pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage, fetchFromGitHub, lib, numpy, pyyaml, cffi, cmake, gcc, git, openblas }:
+buildPythonPackage rec {
+  version = "v0.1.12";
+  name = "pytorch-${version}";
+
+  src = fetchFromGitHub {
+    owner = "pytorch";
+    repo = "pytorch";
+    rev = version;
+    sha256 = "0r8mf4xya76gz83y5z3hfxh0rydkydafhipl8g7d0bfrgw961jy9";
+  };
+
+  checkPhase = ''
+    bash test/run_test.sh  	    
+  '';
+
+  buildInputs = [
+     cmake
+     gcc
```

Comment:
Is `gcc` explicitly necessary? The standard environment provides a compiler. That is not enough?